OpenQA/Isotovideo/Utils.pm

```diff
@@ -128,7 +128,7 @@ configuration variables.
 sub handle_generated_assets ($command_handler, $clean_shutdown) {
     my $return_code = 0;
     # mark hard disks for upload if test finished
-    return unless $bmwqemu::vars{BACKEND} =~ m/^(qemu|generalhw)$/;
+    return unless defined &backend::svirt::do_extract_assets;
```

Comment:
That's now completely ignoring what kind of backend we have :)

Reply:
You're right 🤦🏾 I was thinking to change it but totally forgot.

Reply:
Sorry about the delay. Turns out this way didn't work with the backend object, so I changed it to "can" like you recommended before since I had to change it anyway.

Reply:
> Turns out this way didn't work with the backend object

Can you elaborate on why/how this didn't work?

Reply:
e.g. my suggestion would have been:
```
return if ($bmwqemu::vars{BACKEND} ne `qemu` and
  ($bmwqemu::vars{BACKEND} ne `svirt` or not defined &backend::svirt::do_extract_assets));
```

Reply:
> Can you elaborate on why/how this didn't work?

`&$bmwqemu::backend::backend::do_extract_assets` gives me `Use of uninitialized value $bmwqemu::backend::backend::do_extract_assets in subroutine dereference` and without the deref it doesn't work.

Reply:
That's not what I suggested in the first place, but checking `&backend::svirt::do_extract_assets` **in case** the backend is `svirt`.
Checking something like that with a variable name needs a bit more complicated syntax.

Reply:
Anyway, since you deleted the "not implemented" method in this PR, the `can()` is ok.